src/gitHandlers/pullRequest.ts

```diff
@@ -36,7 +38,13 @@ export const pullRequest = async ({ action, pull_request }: IPullRequestEvent) =
                 })
             }
 
-            if (action === "opened" && ENABLE_PR_QUOTES && prQuotes.coolCats.includes(user.login)) {
+            if (action !== "opened")
+                break;
+
+            let PRUrl = `${GITHUB_URL}repos/${owner}/${name}/pulls/${number}`;
+            postMessage({ text: `@here A New PR is ready for review: ${PRUrl}`, channel: "hippo-devs" });
+
+            if (ENABLE_PR_QUOTES && prQuotes.coolCats.includes(user.login)) {
```

Comment:
Only came here to make sure this part didn't get lost.  Business critical.

Reply:
Its critical. You have to keep it!